### DISCUSSION TITLE
بستن سایت های بد افزار به دلیل گیر دادن دیتاسنتر

### DISCUSSION THREAD

**Discussion by razyar** [Gozargah/Marzban#612] [Q&A]
سلام راه حلی سراغ دارید سایت های کرکینگ و بدافزار رو بدون اینکه دونه دونه بزنیم بشه بست؟ دمتون گرم.

**Comment by Mjzaret** [comment]
به نظرم از وارپ استفاده کن
اینطوری آی پی سرورت نمیافته
منم مشکل ابیوز داشتم که همیشه نت اسکن روی سرورم انجام میشد
برای همین آی پی هایی که دیتا سنتر بهم میداد رو روت میکردم سمت وارپ

**Reply by razyar** [reply]
دمت گرم.
وارپ رو زده بودم، جدا از اون بنظرم بهترین راه حل بستن سایت پورنه. اکثر گیر دادنای دیتاسنتر سر pop up هاییه که تو سایت پورن برا یوزر میاره.
"geosite:category-porn"
رو به اخر کد xray اضافه کنید.
وارپ رو هم بزنین سرعت یکم پایین میاد ولی از بسته شدن بهتره ؛)

**Comment by hawshemi** [comment]
https://github.com/Chocolate4U/Iran-v2ray-rules/
